Doc/whatsnew/3.13.rst

```diff
@@ -97,6 +97,9 @@ Other Language Changes
   if supported.
   (Contributed by Victor Stinner in :gh:`109649`.)
 
+* :func:`os.path.realpath` now resolves MS-DOS style file names even if
+  the file is not accessible.
+  (Contributed by Moonsik Park in :gh:`110298`.)
```

Comment:
This is generally the issue number rather than the PR number, so `82367` rather than `110298`.

Reply:
changed it to the issue number.